src/components/organisms/PaymentModal/PaymentModal.tsx

```diff
@@ -43,18 +42,11 @@ const PaymentModal: React.FunctionComponent<PropsType> = ({
     (state) => state.firestore.status.requested.userPurchaseHistory
   );
 
-  const [isPaymentProceeding, setIsPaymentProceeding] = useState(false);
-  const [isCardBeingSaved, setIsCardBeingSaved] = useState(false);
-
   const hasUserBoughtTicket =
     hasUserBoughtTicketForEvent(purchaseHistory, selectedEvent.id) ||
     (user && isUserAMember(user.email, venue.config?.memberEmails));
 
-  const closePaymentModal = () => {
-    if (!isPaymentProceeding && !isCardBeingSaved) {
-      onHide();
-    }
-  };
+  const closePaymentModal = () => {};
```

Comment:
Why is this still here?

Reply:
fixed in 86ea8ad01c98485b02bc0029048517d83d6b9f3d